tests/integration/pipeline/test_bootstrap_command.py

```diff
@@ -0,0 +1,227 @@
+from unittest import skipIf
+
+from parameterized import parameterized
+
+from tests.integration.pipeline.base import BootstrapIntegBase
+from tests.testing_utils import (
+    run_command_with_input,
+    RUNNING_ON_CI,
+    RUNNING_TEST_FOR_MASTER_ON_CI,
+    RUN_BY_CANARY,
+    run_command,
+)
+
+# bootstrap tests require credentials and CI/CD will only add credentials to the env if the PR is from the same repo.
+# This is to restrict tests to run outside of CI/CD, when the branch is not master or tests are not run by Canary
```

Comment:
I didn't understand this comment

Reply:
We don't want to run these tests in PR because it will expose our AWS creds.

The comment is the same as in 

https://github.com/aws/aws-sam-cli/blob/5cc7680068c820e972d6165a0cccd21677e2a428/tests/integration/deploy/test_deploy_command.py#L19-L21